services/cd-service/pkg/repository/transformer.go

```diff
@@ -869,10 +869,36 @@ func (c *DeployApplicationVersion) Transform(ctx context.Context, state *State)
 	if err := util.WriteFile(fs, manifestFilename, manifestContent, 0666); err != nil {
 		return "", err
 	}
+
+	deployInfo := auth.Extract(ctx).Name + "; " + time.Now().UTC().String() + "\n"
+	deploymentFilename := fs.Join(releaseDir, "environments", c.Environment, "deployment_metadata.yaml")
```

Comment:
Please write the data as dedicated files (one per field) like it's done for locks here: https://github.com/freiheit-com/kuberpult/blob/300a1f54f5818b80a3bb338a5428612658fbcb0a/services/cd-service/pkg/repository/transformer.go#L607C11-L607C11

Reply:
We will store multiple deployments. I think for this case it's more consistent to have one file with multiple entries.
Otherwise we can get into odd edgecases where there's 1 author and 2 timestamps.

(story: https://revolution.dev/app/-JqFGExX46gs9mH7vxR5/-NQPLdkfXUNc4bE9KOiO/FILESYSTEM/FOLDER/ROOT/STORY/-NWiltvEUw2EQt4SdlvI?iteration=CURRENT_ITERATIONS&user=-NHUO4EmpZZtkcSSqr1i)

Alternative: We create a directory with an array (like "releases") and have single files in `deploymentMetadata/1/author` and `deploymentMetadata/1/deploymentTimestamp`

What do you think @hannesg ?